src/resources/postcss/tickets-registration-page.pcss

```diff
@@ -1,8 +1,10 @@
-@import "tickets-common.pcss";
 /**
  * Compatibility
 */
 
+/* Import Mixins */
+@import '_mixins.pcss';
+
```

Comment:
where do the variables come from if we remove this?

Reply:
from common-skeleton and common-full